src/main/java/sirius/biz/tycho/kb/KBHelper.java

```diff
@@ -31,4 +31,8 @@ public KnowledgeBaseArticle currentArticle() {
     protected void installCurrentArticle(KnowledgeBaseArticle article) {
         currentArticle.set(article);
     }
+
+    protected void clearCurrentArticle() {
```

Comment:
Javadoc is missing.

Reply:
Technically protected ;-) - but still documented now^^